public/_redirects

```diff
@@ -0,0 +1 @@
+/* /index.html 200
```

Comment:
This seems good? Why did you add it? What does it do?

Reply:
This is a redirect rule that fixed an issue where navigating to the Netlify deploy preview url resulted in a 404 error. This was happening when trying to reach the preview on the Samsung tablet, Sergio's iPad, and my iPhone. My attempt to fix this led me to these resources.

https://github.com/developit/preact-boilerplate/issues/101
https://www.netlify.com/docs/redirects/